changelog.md

```diff
@@ -11,20 +11,16 @@
 # 2018-06-20 - Add gradient definitions to React Native
 
 **Breaking:**
-
```

Comment:
Staying consistent with the rest of the changelog by removing newlines after "titles"...